frontend_tests/node_tests/compose_ui.js

```diff
@@ -71,6 +73,25 @@ function make_textbox(s) {
     return widget;
 }
 
+run_test("autosize_textarea", () => {
+    const textarea_autosized = {};
+
+    autosize.update = (textarea) => {
```

Comment:
You will want to reset the value of `autosize.update` at the end of the test.  You can look for `with_field` in other node tests to figure out how to use it.

Reply:
Updated. Thanks!